aries-test-harness/features/steps/present-proof-v3.py

```diff
@@ -0,0 +1,317 @@
+from behave import *
+import json
+from agent_backchannel_client import agent_backchannel_POST, expected_agent_state
+from agent_test_utils import (
+    get_relative_timestamp_to_epoch,
+    amend_presentation_definition_with_runtime_data,
+)
+from distutils.util import strtobool
+
+
+@given(
+    '"{issuer}" has issued a credential with {credential_data} to "{prover}" using issue-credential v3'
+)
```

Comment:
This code in this step implementation is very close to what is already in 0454-present-proof-v2.py. Could we not modify that original step to handle both?

Reply:
Done!